src/ontology/mondo-edit.obo

```diff
@@ -447927,7 +447924,6 @@ def: "OBSOLETE. Holmes-Gang syndrome is an X-linked mental retardation (XLMR) sy
 comment: Reason: out of scope. Term to consider: alpha thalassemia-X-linked intellectual disability syndrome'
 subset: ordo_malformation_syndrome {source="Orphanet:93970"}
 xref: ICD10CM:Q87.0 {source="Orphanet:93970/attributed", source="Orphanet:93970/ntbt", source="Orphanet:93970"}
-xref: Orphanet:93970 {source="MONDO:obsoleteEquivalentObsolete"}
```

Comment:
Removed because this was flagged in a proxy merge conflict with `MONDO_0010663` and `MONDO_0019774`.